app/src/main/java/com/babylon/wallet/android/MainActivity.kt

```diff
@@ -54,29 +59,32 @@ class MainActivity : FragmentActivity() {
 
         setContent {
             RadixWalletTheme {
-                CompositionLocalProvider(LocalActionableAddressViewEntryPoint provides actionableAddressViewEntryPoint) {
-                    val isDevBannerVisible by viewModel.isDevBannerVisible.collectAsState(initial = true)
-                    val devBannerState by remember(isDevBannerVisible) {
-                        derivedStateOf { DevBannerState(isVisible = isDevBannerVisible) }
-                    }
+                val isVisible by balanceVisibilityObserver.isBalanceVisible.collectAsState(initial = true)
+                CompositionLocalProvider(LocalBalanceVisibility provides isVisible) {
+                    CompositionLocalProvider(LocalActionableAddressViewEntryPoint provides actionableAddressViewEntryPoint) {
```

Comment:
Really minor: I would suggest these two providers to be put in a composable like
```
@Composable //or a better name
fun WithCustomProviders(content: @Composable () -> Unit { 
    val isVisible by balanceVisibilityObserver.isBalanceVisible.collectAsState(initial = true)
    CompositionLocalProvider(LocalBalanceVisibility provides isVisible) {
        CompositionLocalProvider(LocalActionableAddressViewEntryPoint provides actionableAddressViewEntryPoint) {
            content()
        }
     }
}
```

Reply:
any idea where to place this composable?